next.config.js

```diff
@@ -1,4 +1,8 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
+const nextConfig = {
+    images: {
+        unoptimized: true,
+    }
```

Comment:
Which images are we making unoptimized? Is it just the click logo?

Reply:
My read of [the docs is](https://nextjs.org/docs/pages/api-reference/components/image#unoptimized) that the unoptimized option only applies `<Image>`, which we don't seem to use anywhere.

It was in Vincent's first pass, so I didn't think to check. Thanks for pointing it out. Dropped it.